lib/engine/game/g_18_fr/step/buy_sell_par_shares.rb

```diff
@@ -8,6 +8,18 @@ module G18FR
       module Step
         class BuySellParShares < G1817::Step::BuySellParShares
           MIN_BID = 90
+
+          def corporate_actions(entity)
+            # In Stock Round corporations can only take loans
+            return [] if @winning_bid
+
+            return [] if @corporate_action && @corporate_action.entity != entity
+
+            actions = []
+            actions << 'take_loan' if @round.current_actions.none? && @game.can_take_loan?(entity)
```

Comment:
Don't call `none?` without a block, use `empty?` instead.
```suggestion
            actions << 'take_loan' if @round.current_actions.empty? && @game.can_take_loan?(entity)
```